Tasks/ServiceFabricDeploy/task.json

```diff
@@ -150,6 +141,15 @@
             "groupname": "advanced",
             "helpMarkDown": "Indicates whether the package should be validated or not before deployment."
         },
+        {
```

Comment:
not comfortable making this part of this hotfix
you should remove diff related changes